README.md

```diff
@@ -23,8 +23,8 @@ This project is intended to be used with the latest Active LTS release of [Node.
 To clone the repository, use the following commands:
```

Comment:
In next iterations, this will likely get to the bottom of the README and be changed by a `npm i @aragon/cli` type of command. However, always good to show people how to clone and use the repo locally.

Feel free to resolve comment, but thought I'd bring this up.